<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgAddGraphCurve</class>
 <widget class="QDialog" name="CQDlgAddGraphCurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Curve</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>X-value =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-value =</string>
   </property>
  </widget>
  <widget class="QLabel" name="qqZValue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-value =</string>
   </property>
  </widget>
  <widget class="QComboBox" name="qqComboX">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="qqComboY">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="qqComboZ">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>80</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
